### DISCUSSION TITLE
cant seem to get drives in machinaris seen by smartctl.

### DISCUSSION THREAD

**Discussion by danktankk** [guydavis/machinaris#909] [Q&A]
I have added the required parts for the compose file explained in the Wiki, however nothing is populating under this area.

I bashed into the machinaris container to see what smartctl says about /dev/sdb for example, and this is what I am getting back:

![image](https://github.com/guydavis/machinaris/assets/34148516/1595e93b-ab6c-45e1-a2f1-22143a2977b7)

Im not sure that I can use this functionality the way I am running machinaris, but I wanted to at least check to be sure.  Thanks for any help.  I have machinaris running in a proxmox server VM running ubuntu server 22.04.  The drives are passed through so they can be read by the container.

**Comment by guydavis** [comment]
Greetings, please take a look at [overriding](https://github.com/guydavis/machinaris/wiki/Drives#override-command) the command, per-drive, that Machinrais uses to query for `smartctl` status. Hope this helps.
